internal/config/config.go

```diff
@@ -50,6 +50,9 @@ const (
 
 // Bind binds the actual data from the receiver field.
 func (c *GlobalConfig) Bind() *GlobalConfig {
+	if c == nil {
+		c = new(GlobalConfig)
+	}
```

Comment:
you should create the test for covering these codes.
I'm not sure this code is correct?
I think when `c==nil`, `Bind()` method cannot be running 🤔

Reply:
thank you.

> I think when c==nil, Bind() method cannot be running 🤔
got it, I think so.
I think we should delete it

Reply:
fixed: https://github.com/vdaas/vald/pull/1219/commits/760d84928e84d06bc026398dffcce0c088d7e49c